waspc/data/Generator/templates/sdk/wasp/server/auth/user.ts

```diff
@@ -63,6 +63,23 @@ export type AuthEntityWithIdentities = {= authEntityName =} & {
   {= identitiesFieldOnAuthEntityName =}: {= authIdentityEntityName =}[]
 }
 
+// PRIVATE API
```

Comment:
This is basically a copy of the types we used so far, but it reduces the needed fields from _all the AuthIdentity fields_ to only the ones we really need.